core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -838,6 +838,13 @@ oppia.factory('explorationLanguageCodeService', [
         }
       }
     };
+    child.getLanguageDescription = function(languageCode) {
```

Comment:
I don't think this has anything to do with this factory, actually. You might want to define a separate utility factory for handling global language-related operations that aren't exploration-specific.